### DISCUSSION TITLE
Error trying to rescale

### DISCUSSION THREAD

**Discussion by VernonDev01** [Coyote-A/ultimate-upscale-for-automatic1111#57] [Q&A] [answered]
Hello, I would like to know why when I try to resize an image I get this error; any solution that you can give me please:

RuntimeError: Given groups=1, weight of size [64, 3, 3, 3], expected input[1, 4, 192, 192] to have 3 channels, but got 4 channels instead

**Comment by Jopezzia** [comment, marked as answer]
Hi!

Check this:
https://github.com/Coyote-A/ultimate-upscale-for-automatic1111/wiki/FAQ#i-have-something-like-runtimeerror-given-groups1-weight-of-size-64-3-3-3-expected-input1-4-256-256-to-have-3-channels-but-got-4-channels-instead-error

**Reply by VernonDev01** [reply]
Oh, what was that about, thank you very much for taking the trouble to help me; Peace!
